packages/connect-core/src/utils/path/describePath.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-empty */
+import { ethers } from 'ethers'
+
+import Application from '../../entities/Application'
+import {
+  tryEvaluatingRadspec,
+  postprocessRadspecDescription,
+} from '../radspec/index'
+import { TransactionRequestData } from '../../transactions/TransactionRequest'
+
+/**
+ * Use radspec to create a human-readable description for each transaction in the given `path`
+ *
+ */
+export async function describeTransactionPath(
```

Comment:
This function is only considering the straight forward Radspec implementation. Here we will include a couple more handlers when we improve our transaction path support.